src/test/scala/com/microsoft/ml/spark/cognitive/split1/TextAnalyticsSDKSuite.scala

```diff
@@ -1,35 +1,73 @@
 package com.microsoft.ml.spark.cognitive.split1
 
 import com.azure.ai.textanalytics.models.TextAnalyticsRequestOptions
+import com.microsoft.ml.spark.Secrets
 import com.microsoft.ml.spark.cognitive._
 import com.microsoft.ml.spark.core.test.base.TestBase
+import com.microsoft.ml.spark.core.test.fuzzing.TransformerFuzzing
 import org.apache.spark.ml.param.DataFrameEquality
-import org.apache.spark.sql.DataFrame
+import org.apache.spark.sql.{DataFrame, Row}
+import org.apache.spark.sql.functions.col
 
 class TextAnalyticsSDKSuite extends TestBase with DataFrameEquality with TextKey {
   import spark.implicits._
-
   lazy val df: DataFrame = Seq(
     "Hello World",
     "Bonjour tout le monde",
     "La carretera estaba atascada. Había mucho tráfico el día de ayer.",
-    ":) :( :D"
+    "世界您好",
+    ":) :( :D",
   ).toDF("text2")
 
   val options: Option[TextAnalyticsRequestOptions] = Some(new TextAnalyticsRequestOptions()
     .setIncludeStatistics(true))
 
   lazy val detector: TextAnalyticsLanguageDetection = new TextAnalyticsLanguageDetection(options)
     .setSubscriptionKey(textKey)
-    .setEndpoint("https://eastus.api.cognitive.microsoft.com/")
+    .setEndpoint("https://ta-internshipconnector.cognitiveservices.azure.com/")
     .setInputCol("text2")
 
   test("Basic Usage") {
     val replies = detector.transform(df)
       .select("name", "iso6391Name")
       .collect()
+//      .select(detector.getPrimaryDetectedLanguageNameCol)
+//      .collect().toList
 
-    assert(replies(0).getString(0) == "English" && replies(2).getString(0) == "Spanish")
+    assert(replies(0).getString(0) == "English" && replies(2).getString(0) == "Spanish" )
+    assert(replies(3).getString(0) == "Chinese_Traditional")
     assert(replies(0).getString(1) == "en" && replies(2).getString(1) == "es")
+
+  }
+}
+
+class TextSentimentSuiteV4 extends TestBase with DataFrameEquality with TextKey {

```

Comment:
Creating a new class just for sentiment is great at help organize things. Make sure that other tests (like LD) and @preetipidatala KPE PR follow the same pattern.